tests/codes/test_stabilizer_code.py

```
@@ -0,0 +1,409 @@
+    assert not c1.equal_stabilizer_group(c2)
+    assert not c1.equal_logical_basis(c2)
+    assert c1.get_logical_mapping(c2) is None
+
+
+def test_logical_operator_checks() -> None:
+    """Test checks for logical and stabilizer operators."""
+    c1 = StabilizerCode(["XXXX", "ZZZZ"])
+    p_str = "XIII"
+    p = Pauli.from_pauli_string(p_str)
+
+    assert c1.is_z_logical(p) == c1.is_z_logical(p_str) == False
```

Ruff (0.15.21)

[error] Avoid equality comparisons to `True` or `False`

Replace comparison

(E712)